packages/synapse-react-client/src/lib/containers/auth/TwoFactorBackupCodes.tsx

```diff
@@ -16,16 +15,12 @@ import IconSvg from '../IconSvg'
 import { useMutation } from 'react-query'
 import { createRecoveryCodes } from '../../utils/SynapseClient'
 import { useSynapseContext } from '../../utils/SynapseContext'
-import TwoFactorGraphic from '../../assets/icons/twofactor-graphic.svg'
 import { displayToast } from '../ToastMessage'
 import { RegenerateBackupCodesWarning } from './RegenerateBackupCodesWarning'
 import { RecoveryCodeGrid } from './RecoveryCodeGrid'
-
-const hideOnPrintStyle: SxProps = {
```

Comment:
Hiding certain elements didn't totally work because things like the navbar end up getting printed